paraflow-metaserver/src/main/java/cn/edu/ruc/iir/paraflow/metaserver/client/MetaClient.java

```diff
@@ -53,21 +55,55 @@ public void shutdownNow()
         this.channel.shutdownNow();
     }
 
+    private boolean nameFormat(String userName)
+    {
+        String regEx = "^[a-zA-Z][a-zA-Z0-9]*$"; //表示a或f
+        Pattern p = Pattern.compile(regEx);
+        Matcher m = p.matcher(userName);
+        return m.find();
+    }
+
+    private boolean length(String name)
+    {
+        int length = name.length();
+        boolean len = true;
+        if (length > 50) {
+            len = false;
+        }
+        return len;
+    }
+
+    private boolean urlFormat(String url)
+    {
+        String regEx = "^[a-zA-Z]+://[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{1,5}(/[a-zA-Z]+)+$"; //表示a或f
```

Comment:
host name may not contain ip address, for example:
hdfs://localhost:9000/example